docs/local_development.md

```diff
@@ -44,16 +44,16 @@ docker images
 1. Set the `repository` and `tag` values of your local image (notice that the `IMAGE ID` might be different in your local installation)
 
 ```
-docker image tag 85b0eb016889 princeton_dev:latest
+docker image tag 85b0eb016889 mediaflux_dev:latest
```

Comment:
@bess I'm curious why you changed the tag.  The rest of us had this tag previously...

Reply:
Changed because "princeton" is not the name of an application and I thought "mediaflux_dev" was a better name. (I know I said this on slack already; just repeating it here for posterity)